Mvc/Olive.Email/Olive.Email.csproj

```diff
@@ -29,7 +29,7 @@
       <HintPath>..\..\@Assemblies\netstandard2.0\Olive.Entities.dll</HintPath>
     </Reference>
     <Reference Include="Olive.Web">
-      <HintPath>..\..\@Assemblies\netstandard2.0\Olive.Web.dll</HintPath>
```

Comment:
Olive.MVC is .net core app and this reference is not a good reference, while it will be available there because it's a dependancy of the other projects. but, default generated path it that one.